src/api/app/jobs/full_text_search_job.rb

```diff
@@ -0,0 +1,23 @@
+class FullTextSearchJob < ApplicationJob
```

Comment:
*nitpick*: This is not about searching but indexing :)

Reply:
Renamed.